go2_robot_sdk/go2_robot_sdk/infrastructure/sensors/camera_config.py

```
@@ -2,153 +2,269 @@
+    Get a LidarDecoder instance.
+
+    Returns:
+        Initialized LidarDecoder (the working implementation)
+    """
+    return LidarDecoder()
+
+
+def decode_lidar_data(
+    compressed_data: bytes,
+    resolution: float = 0.01,
+    origin: list = [0.0, 0.0, 0.0],
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)